src/features/click-to-load.js

```diff
@@ -1806,7 +1800,20 @@ export default class ClickToLoad extends ContentFeature {
         // Request the current state of Click to Load from the platform.
         // Note: When the response is received, the response handler resolves
         //       the readyToDisplayPlaceholders Promise.
-        sendMessage('getClickToLoadState')
+        const clickToLoadState = await this.messaging.request('getClickToLoadState')
+        this.onClickToLoadState(clickToLoadState)
```

Comment:
Could we use `.then()` with an arrow function here instead? I figure we can inline the `onClickToLoadState` function here as an arrow function.

Reply:
None of the features init calls block on each other if that's the concern?
Otherwise the current code is more readable imo.

Reply:
Fair enough, also I just remembered this message will sometimes be sent unilaterally in the future so Fran's approach here is better too.